doc/api/buffer.md

```diff
@@ -3768,6 +3768,18 @@ console.log(buf2.toString(undefined, 0, 3));
 // Prints: té
 ```
 
+For encoding and decoding large strings to/from `utf-8`,
```

Comment:
What string length is large? Hundreds of bytes, kilobytes, megabytes of text?

And why do we recommend that users switch? Can we not match the performance `TextDecoder` here?

Reply:
I think there is plan to improve perf of buffer.toString() i would love to take that up but so far havent been able to explore that

As for the length of string i guess the length is around 1024 as per https://github.com/nodejs/performance/issues/18#issuecomment-1318875230